backend/src/v4/routes/maps.js

```diff
@@ -614,122 +635,96 @@ function getHereBaseInfo(req, res) {
-	const domain = (1 + ((req.params.gridx + req.params.gridy) % 4)) + hereBaseDomain;
-	const uri = "/maptile/2.1/maptile/newest/normal.day/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/" + size + "/png8";
+	const domain = hereBaseDomain;
+	const uri = "/v3/base/mc/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/png8";
 	systemLogger.logInfo("Fetching Here map tile: " + uri);
 	requestHereMapTile(req, res, domain, uri);
 }
 
 function getHereAerialMapsTile(req, res) {
-	const size = 256; // 256 = [256,256]; 512 = [512,512]; Deprecated: 128
-	const domain = (1 + ((req.params.gridx + req.params.gridy) % 4)) + hereAerialDomain;
-	const uri = "/maptile/2.1/maptile/newest/satellite.day/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/" + size + "/png8";
+	const domain = hereBaseDomain;
+	const uri = "/v3/base/mc/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/png8";
+	req.query.style = "satellite.day";
 	systemLogger.logInfo("Fetching Here map tile: " + uri);
 	requestHereMapTile(req, res, domain, uri);
 }
 
 function getHereTrafficTile(req, res) {
-	const size = 256; // 256 = [256,256]; 512 = [512,512]; Deprecated: 128
-	const domain = (1 + ((req.params.gridx + req.params.gridy) % 4)) + hereTrafficDomain;
-	const uri = "/maptile/2.1/traffictile/newest/normal.day/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/" + size + "/png8";
+	const domain = hereBaseDomain;
+	const uri = "/v3/base/mc/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/png8";
+	req.query.style = "logistics.day";
 	systemLogger.logInfo("Fetching Here traffic flow map tile: " + uri);
 	requestHereMapTile(req, res, domain, uri);
 }
 
 function getHereTrafficFlowTile(req, res) {
-	const size = 256; // 256 = [256,256]; 512 = [512,512]; Deprecated: 128
-	const domain = (1 + ((req.params.gridx + req.params.gridy) % 4)) + hereTrafficDomain;
-	const uri = "/maptile/2.1/flowtile/newest/normal.traffic.day/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/" + size + "/png8";
+	const domain = hereTrafficDomain;
+	const uri = "/v3/flow/mc/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/png8";
 	systemLogger.logInfo("Fetching Here traffic flow map tile: " + uri);
 	requestHereMapTile(req, res, domain, uri);
 }
 
 function getHereTerrainTile(req, res) {
-	const size = 256; // 256 = [256,256]; 512 = [512,512]; Deprecated: 128
-	const domain = (1 + ((req.params.gridx + req.params.gridy) % 4)) + hereAerialDomain;
-	const uri = "/maptile/2.1/maptile/newest/terrain.day/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/" + size + "/png8";
+	const domain = hereBaseDomain;
+	const uri = "/v3/base/mc/" + req.params.zoomLevel + "/" + req.params.gridx + "/" + req.params.gridy + "/png8";
```

Comment:
use the template literals syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals